src/Tokenizer/Tokens.php

```diff
@@ -828,6 +846,21 @@ public function getNextMeaningfulToken(int $index): ?int
         return $this->getMeaningfulTokenSibling($index, 1);
     }
 
+    /**
+     * Require index for closest next token that is not a whitespace or comment.
+     *
+     * @param int $index token index
+     */
+    public function requireNextMeaningfulToken(int $index): int
+    {
+        $index = $this->getMeaningfulTokenSibling($index, 1);
+        if (null === $index) {
+            throw IndexNotFoundException::fromMethod(__FUNCTION__);
+        }
+
+        return $index;
+    }
```

Comment:
with php 8 we could write:

```suggestion
    public function requireNextMeaningfulToken(int $index): int
    {
        return $this->getMeaningfulTokenSibling($index, 1) 
            ?? throw IndexNotFoundException::fromMethod(__FUNCTION__);
    }
```